tracing-subscriber/src/fmt/timing.rs

```diff
@@ -0,0 +1,155 @@
+//! A subscriber that tracks the time spent in each span.
+
+use core::time::Duration;
+use std::time::Instant;
```

Comment:
I wonder whether it's worth considering using [quanta](https://crates.io/crates/quanta) here instead of std::time::Instant?